### PR TITLE
Jpaserver changes

### PR DESCRIPTION
**Pull request by tadgh**
* Make ValueSetOperationProvider not require a fulltext svc.

### CODE REVIEW

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/provider/ValueSetOperationProvider.java [hunk 1]
```diff
@@ -125,8 +125,11 @@ public IBaseResource expand(
 			ValueSetAutocompleteOptions options = ValueSetAutocompleteOptions.validateAndParseOptions(myDaoConfig, theContext, theFilter, theCount, theId, theUrl, theValueSet);
 			startRequest(theServletRequest);
 			try {
-
-				return myFulltextSearch.tokenAutocompleteValueSetSearch(options);
+				if (myFulltextSearch == null || myFulltextSearch.isDisabled()) {
+					throw new InvalidRequestException("Autocomplete is not supported on this server, as the fulltext search service is not configured.");
```
Comment (JasonRoberts-smile) [inline hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/provider/ValueSetOperationProvider.java:129]: chore: this error message should have a Msg number assigned to it

File: hapi-fhir-jpaserver-mdm/src/main/java/ca/uhn/fhir/jpa/mdm/config/MdmSubscriptionLoader.java [hunk 2]
```diff
@@ -87,6 +90,10 @@ synchronized public void daoUpdateMdmSubscriptions() {
 		for (IBaseResource subscription : subscriptions) {
 			updateIfNotPresent(subscription);
 		}
+		//After loading all the subscriptions, sync the subscriptions to the registry.
+		if (subscriptions != null && subscriptions.size() > 0) {
+//			mySubscriptionLoader.syncSubscriptions();
```
Comment (JasonRoberts-smile) [inline hapi-fhir-jpaserver-mdm/src/main/java/ca/uhn/fhir/jpa/mdm/config/MdmSubscriptionLoader.java:95]: question: was this commented out by mistake? If not, what's the point of this if block at all?

**Review by JasonRoberts-smile — APPROVED** [review]
Approved with comments

File: hapi-fhir-jpaserver-mdm/src/main/java/ca/uhn/fhir/jpa/mdm/config/MdmSubscriptionLoader.java (hunk 2 above)
Comment (tadgh) [reply hapi-fhir-jpaserver-mdm/src/main/java/ca/uhn/fhir/jpa/mdm/config/MdmSubscriptionLoader.java:95]: Ah you make a good point. Yes this was commented out by mistake. Another branch was merged into this one and I didn't cleanly remove it. Fixing

**Pull request merged**